images/base/files/usr/local/bin/entrypoint

```diff
@@ -204,6 +204,10 @@ fix_cgroup() {
   # See: https://man7.org/linux/man-pages/man7/cgroups.7.html
   local current_cgroup
   current_cgroup=$(grep -E '^[^:]*:([^:]*,)?cpu(,[^,:]*)?:.*' /proc/self/cgroup | cut -d: -f3)
+  # on WSL2 systems /sys/fs/cgroup/systemd dir is missing which eventually leads to kubelet failing to start.
```

Comment:
Is it WSL2 or just distros without systemd?
Are there WSL2 distros with systemd?

Reply:
At this stage I'm only certain about WSL2. I can reword this to say  "and potentially other distros without systemd"?

Reply:
SGTM